app/components/account/TokenAccountSection.tsx

```diff
@@ -214,7 +214,7 @@ function FungibleTokenMintAccountCard({
                         <td>Bridged Asset Contract</td>
                         <td className="text-lg-end">
                             <Copyable text={assetContractAddress}>
-                                <a href={tokenInfo.extensions.bridgeContract} target="_blank" rel="noreferrer">
+                                <a href={tokenInfo.extensions.assetContract} target="_blank" rel="noreferrer">
```

Comment:
Why is this getting changed?

Reply:
I've made this as it looks like an error. Here: https://github.com/solana-foundation/explorer/pull/567/files/7f0abbfc7c6a953d0ee6a82712fa42cd4c5b239c#diff-3929de3da0169c607b8c0b8f1136a4d32900e470c3af9dcd6cec057578645588L212 the condition checks for assetContract.

Reply:
Master has actual version already. Closing.